testbed/tests/data.py

```diff
@@ -4,20 +4,22 @@
 
 
 COLORS = [
+    # Avoid using (0, 0, 0, 0.0) as a test color,
+    # as that is indistinguishable from the TRANSPARENT color
     rgba(r, g, b, a)
-    for r, g, b in [
+    for r, g, b, a in [
         # Black, gray, white,
-        (0, 0, 0),
-        (1, 1, 1),
-        (10, 10, 10),
-        (128, 128, 128),
-        (245, 245, 245),
-        (254, 254, 254),
-        (255, 255, 255),
+        (0, 0, 0, 1.0),
+        (128, 128, 128, 1.0),
+        (255, 255, 255, 1.0),
         # Primaries
-        (255, 0, 0),
-        (0, 255, 0),
-        (0, 0, 255),
+        (255, 0, 0, 1.0),
+        (0, 255, 0, 1.0),
+        (0, 0, 255, 1.0),
+        # Color with different channel values, including transparency
+        (50, 128, 200, 0.0),
+        (50, 128, 200, 0.5),
+        (50, 128, 200, 0.9),
+        (50, 128, 200, 1.0),
```

Comment:
I've reduced the multiplicity of colours here as it didn't seem to be adding anything to test rigor, but it *was* difficult to see the "meaningful" colors appear in the test series.